.github/workflows/main.yml

```diff
@@ -221,13 +221,13 @@ jobs:
           docker logs $(docker ps | grep front | cut -f 1 -d ' ') > logs/front.log
       - name: Upload test results
         if: always()
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: playwright-results
           path: ./tests/sanity/playwright-report/
       - name: Get Allure history
         uses: actions/checkout@v4
-        if: ${{ github.ref == 'refs/heads/main' }}
+        if: ${{ github.ref == 'refs/heads/main' }} && always()
```

Comment:
Nope, it should be
```suggestion
        if: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v') }}
```

Reply:
great point, thank you!
Also need to add status checks, doing it =)